### PR TITLE
Update 04_study.md

### PR DESCRIPTION
**Pull request by j3xugit**
Revised "protein secondary and tertiary structure" a little bit.
Any comments are welcome.

### CODE REVIEW

File: sections/04_study.md [hunk 1]
```diff
@@ -503,49 +502,44 @@ conformation of a protein structure.
...
-Recently, Wang et al. proposed the deep learning method RaptorX-Contact
+[@doi:10.1093/bioinformatics/btv472] have shown some advantage over pure co-evolution analysis for
+proteins with few sequence homologs, but their accuracy is still far from satisfactory. In
+recent years, deeper architectures have been explored for contact prediction, 
+such as CMAPpro [@doi:10.1093/bioinformatics/bts475], DNCON [@doi:10.1093/bioinformatics/bts598]
+and PConsC [@doi:10.1371/journal.pcbi.1003889]. 
+However, blindly tested in the well-known CASP competitions, 
+these methods did not show any advantage over MetaPSICOV [@doi:10.1093/bioinformatics/btu791].
+
+Very recently, Wang et al. proposed the deep learning method RaptorX-Contact
```
Comment (agitter) [inline sections/04_study.md]: It's not too important to me, but I suggested removed "Very" during our last review.  Someone reading this in 2018 (or later) may think a method that was pre-printed in 2016 isn't "very" recent any more.  Do you care if we remove it?

File: sections/04_study.md [hunk 2]
```diff
@@ -503,49 +502,44 @@ conformation of a protein structure.
 
 Protein contact prediction and contact-assisted folding (i.e. folding proteins
 using predicted contacts as restraints) represents a promising new direction for
-ab initio folding of proteins without good templates in PDB. Evolutionary
-coupling analysis (ECA) is effective for proteins with a very large number
-(>1000) of sequence homologs [@doi:10.1371/journal.pone.0028766], but otherwise
-fares poorly for proteins without many sequence homologs. By combining ECA with
-a few other protein features, shallow neural network-based methods such as
+ab initio folding of proteins without good templates in PDB. Co-evolution analysis
```
Comment (agitter) [inline sections/04_study.md]: `*ab initio*`

File: sections/04_study.md [hunk 3]
```diff
@@ -503,49 +502,44 @@ conformation of a protein structure.
...
-trained mostly on non-membrane proteins [@arxiv:1704.07207]. RaptorX-Contact
+This technique also worked well on membrane proteins even when
+trained on non-membrane proteins [@arxiv:1704.07207]. RaptorX-Contact
 performed better mainly due to introduction of residual neural networks and
-exploitation of contact occurrence patterns by simultaneously predicting of all
+exploitation of contact occurrence patterns by simultaneously predicting all
 the contacts in a single protein.
 
-Taken together, we believe it is still possible to further improve contact
+Taken together, ab initio folding is becoming much easier with the advent
```
Comment (agitter) [inline sections/04_study.md]: `*ab initio*`

File: sections/04_study.md (hunk 1 above)
Comment (j3xugit) [inline sections/04_study.md]: yep, please remove "very".

**Pull request merged**